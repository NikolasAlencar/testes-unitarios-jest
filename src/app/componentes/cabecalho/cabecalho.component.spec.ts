import { ComponentFixture, TestBed } from '@angular/core/testing';
import { CabecalhoComponent } from './cabecalho.component';

describe('CabecalhoComponent', () => {
  let component: CabecalhoComponent;
  let fixture: ComponentFixture<CabecalhoComponent>;

  beforeEach(() => {
    TestBed.configureTestingModule({
      imports: [CabecalhoComponent],
    });

    fixture = TestBed.createComponent(CabecalhoComponent);
    component = fixture.componentInstance;
  });

  it('should be created', () => {
    expect(component).toBeTruthy();
  });

  it('should define the inputs alt and src', () => {
    expect(component.alt).toBeDefined();
    expect(component.src).toBeDefined();
  });

  it('should render the content based in inputs alt and src', () => {
    component.alt = 'Teste';
    component.src = 'https://teste.com/teste.jpg';
    expect(component).toMatchSnapshot();
  });
});
