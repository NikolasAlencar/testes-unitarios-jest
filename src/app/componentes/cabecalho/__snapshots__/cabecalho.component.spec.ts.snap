// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CabecalhoComponent should render the content based in inputs alt and src 1`] = `
CabecalhoComponent {
  "__ngContext__": 4,
  "alt": "Teste",
  "src": "https://teste.com/teste.jpg",
}
`;
